testsuite/objects/__init__.py

```diff
@@ -150,14 +155,19 @@ def add_rule(self, when: list[Rule]):
 class PreexistingAuthorino(Authorino):
     """Authorino which is already deployed prior to the testrun"""
 
-    def __init__(self, authorization_url, oidc_url) -> None:
+    def __init__(self, authorization_url, oidc_url, metrics_url) -> None:
         super().__init__()
         self._authorization_url = authorization_url
         self._oidc_url = oidc_url
+        self._metrics_url = metrics_url
 
     def wait_for_ready(self):
         return True
 
+    @property
+    def metrics_service(self):
+        return self._metrics_url.split(".", 1)[0]
```

Comment:
if you need metrics url in specific format, why dont you request that format in the settings directly?

Reply:
It is an option. The point was to request all services in the same format so that it won't be a little confusing... https://github.com/Kuadrant/testsuite/blob/c76fcea6d8501c0843e1b23e562dfbca7531b08d/config/settings.local.yaml.tpl#L35-L37

Reply:
What is the expected format?

Reply:
Service URL look like this `<service>.<pod_namespace>.svc.cluster.local`. I need only a service name to label this service properly so that *ServiceMonitor* can discover it.

Reply:
Why not ask only for service_name and name the variable appropriately, not naming it "URL"